packages/core/components/Footnotes/footnotes.css

```diff
@@ -0,0 +1,5 @@
+:is(ul) {

```

Comment:
Should this have an outer wrapper on it? Maybe cdc-open-viz-module or a footnotes class?